py/server/deephaven/pandas.py

```diff
@@ -85,7 +111,8 @@ def _column_to_series(table: Table, col_def: Column) -> pd.Series:
 }
 
 
-def to_pandas(table: Table, cols: List[str] = None, dtype_backend: str = None) -> pd.DataFrame:
+def to_pandas(table: Table, cols: List[str] = None, dtype_backend: str = None, conv_null: bool = False) -> \
```

Comment:
I have conflicting feelings:
1. Making conv_null default to True offers better user experiences
2. But it will be a breaking change. And it creates Python nullable extension arrays even if the data doesn't contain any nulls thus consumes more memory. 

I am leaning towards not making it the default, but can go the other way too.

Reply:
I'm leaning to defaulting to `pd.NA`.  I think that default results in the least surprise for a user.   Is the extra memory enough to change my thinking?

@jjbrosnan and @alexpeters1208 should also comment.

Reply:
Again, depends on what a python user expects. If this conversion happens by default for other objects that get converted to Pandas, I think this should do that as well.

Reply:
Pandas (and Python) users typically prefer ease-of-use, even if it costs some extra memory. IMO the default should be set to True, with a note about memory use. I don't completely understand how that would be a breaking change, though. What would it break?

Reply:
A Pandas nullable extension type such as `pd.Int64Dtype` uses a numpy bool array as the mask to indicate which elements are pd.NA. So the extra memory usage is roughly array_size * 1 byte. 

It is a breaking change because the result dtype won't be a np.dtype such as `np.int64` but instead `pd.Int64Dtype()`.